scripts/getToken.js

```diff
@@ -0,0 +1,20 @@
+const tokenRequester = require('keycloak-request-token')
+
+const baseUrl = 'http://localhost:8080/auth';
```

Comment:
What is this about?

Reply:
TL;DR it's a development tool to make consuming the examples easier

This file will be a simple helper script to get the `{Authorization: 'Bearer ...'}` value that we paste into GraphQL playground. This is a much simpler and developer friendly way to get that token without potential security issues. (no more `/token`) endpoint.

The reason it's needed is that when using the `keycloak.middleware()` instead of `keycloak.protect()`, unauthenticated users will not get redirected to the keycloak login page when they try to access the playground.

Reply:
Yes. This is what I was thinking. The playground gives the ability to actually pass this value into the window - it will mean refresh but the overal experience will be much better. Integrating keycloak with Playground is really cool stuff to have

Reply:
Last time I checked, I couldn't find a way to do that but I will take a look again for sure!